canary/mqtt5/canary.ts

```diff
@@ -13,62 +13,111 @@ type Args = { [index: string]: any };
 const yargs = require('yargs');
 
 yargs.command('*', false, (yargs: any) => {
-    yargs.option('duration', {
-        description: 'INT: time in seconds to run the canary',
-        type: 'number',
-        default: 3600,
-    })
+    yargs.option({
+        'duration': {
+            description: 'INT: time in seconds to run the canary',
+            type: 'number',
+            default: 120,
+        },
+        'endpoint': {
+            description: 'STR: endpoint to connect to',
+            type: 'string',
+            default: 'localhost',
+        },
+        'port': {
+            description: 'number: port to connect to',
```

Comment:
nit: duration option uses `INT` word in the description, while other numeric options use `number`. It's better to use one term, for consistency.